arcgis-ios-sdk-samples/Content Display Logic/Controllers/OptionsTableViewController.swift

```diff
@@ -95,6 +96,11 @@ class OptionsTableViewController: UITableViewController {
             }
             tableView.reloadRows(at: indexPathsToReload, with: .automatic)
             onChange(indexPath.row)
+        } else if allowsEmptySelection {
+            tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
```suggestion
```
Since the row is reloaded after the `selectedIndex` being nullified, no need to deselect the row. However, it might be good to keep it in terms of parity with the other conditional branches.